semantic-kernel-python/run.sh

```diff
@@ -7,42 +7,28 @@ set -x
 CONN_STRING=$(fetch_local_atlas_uri)
 PYTHON_BINARY=$(find_python3)
 
-
 # WORKING_DIR = src/semantic-kernel-python/semantic-kernel
 cd python
 
+$PYTHON_BINARY -m venv .
```

Comment:
Do you mind giving the virtual environment it's own name? Just best practice.

Reply:
That command would put the venv in the CWD, so maybe `PYTHON_BINARY -m venv .venv` ?

Reply:
`make install-sk ` appears to create `.venv`.

Reply:
nice catch. I don't need this line. I think the `uv pytest ...` call will run the tests within the virtual environment

Reply:
turns out I do 😢 (for now). https://github.com/microsoft/semantic-kernel/issues/9067

Reply:
`uv` is on PyPI in case that helps. I use `pipx` to install it in [my dev setup](https://github.com/aclark4life/setup?tab=readme-ov-file#pipx).

Reply:
Yeah, the workaround just has me doing the standard issue pip install. It just requires I wrap it in another virtualenv to make sure I can call the subsequent uv commands.